presto-main/src/test/java/io/prestosql/operator/scalar/timestamp/TestTimestampLegacy.java

```diff
@@ -13,11 +13,41 @@
  */
 package io.prestosql.operator.scalar.timestamp;
 
+import io.prestosql.Session;
+import org.testng.annotations.Test;
+
+import static io.prestosql.spi.type.TimeZoneKey.getTimeZoneKey;
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class TestTimestampLegacy
         extends BaseTestTimestamp
 {
     protected TestTimestampLegacy()
     {
         super(true);
     }
+
+    @Test
+    public void testCastToTimeWithDaylightSavings()
+    {
+        // The number of seconds since the beginning of the day for the America/Los_Angeles in 2020 doesn't match
+        // that of 1970, the year of the epoch. Make sure the proper corrections are being performed.
+        Session session = assertions.sessionBuilder()
+                .setTimeZoneKey(getTimeZoneKey("America/Los_Angeles"))
```

Comment:
nit: We usually use Kathmandu for this.

Reply:
Does it matter? This timezone reproduces the issue and is easy to reason about.

Reply:
Kathmandu zone is very easy to reason about.  It doesn't

Reply:
I guess what I'm referring to is to the fact that while I know what the daylight saving rules for America/Los_Angeles, I have no idea about Kathmandu's. I'd need to look them up (what day it switches over, etc).

Reply:
We use Kathmandu in the code base for two reasons:
- it has an offset that is not whole hours (x:45)
- it has a different offset on 1970-01-01 than "today" 
(and it has no DST btw, which probably doesn't matter for this test)

https://github.com/prestosql/presto/blob/4e1393d4933f73876999944ef9c7d0b5ddd3e534/presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java#L357
https://github.com/prestosql/presto/blob/21e3ddf55a641e1c17b7e4a17689223ff1120b42/presto-main/src/test/java/io/prestosql/operator/scalar/TestDateTimeFunctionsBase.java#L196
https://github.com/prestosql/presto/blob/21e3ddf55a641e1c17b7e4a17689223ff1120b42/presto-main/src/test/java/io/prestosql/operator/scalar/TestDateTimeFunctionsBase.java#L1125-L1126

That doesn't not surprise me you know the rules for Los Angeles. I would need to look them up though.

To be clear: I am fine with this test using Los Angeles zone. 
I just do not agree that Los Angeles is "better" in general.
(See also https://github.com/prestodb/presto/issues/10078)

Reply:
Of course, we do not need (and we should not) use 2-4 selected zones everywhere.